hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java

```diff
@@ -3607,7 +3607,7 @@ private Block addStoredBlock(final BlockInfo block,
       curReplicaDelta =
           (node.isDecommissioned() || node.isDecommissionInProgress()) ? 0 : 1;
       if (logEveryBlock) {
-        blockLog.debug("BLOCK* addStoredBlock: {} is added to {} (size={})",
+        blockLog.info("BLOCK* addStoredBlock: {} is added to {} (size={})",
```

Comment:
+1. Please double check if it could flood general log in some corner case.

Reply:
Thanks @Hexiaoqiao for your review.  We have released it in our prod environment and it works well.

`logEveryBlock=true` means that we should output some logs about this block. And I have checked the callers and they are expected.